requirements_test.txt

```diff
@@ -9,7 +9,7 @@ pre-commit==2.15.0
 pycodestyle==2.7.0
 pyflakes==2.3.1
 flake8-docstrings==1.6.0
-pydocstyle==6.0.0
+pydocstyle==6.1.1
```

Comment:
Side note: If we use flake8-docstrings we shouldn't also need pydocstyle. We should consolidate our CI checks to use the former.